app/src/main/java/live/hms/app2/ui/meeting/participants/RtmpRecordFragment.kt

```diff
@@ -192,4 +186,8 @@ class RtmpRecordFragment : Fragment() {
         // Actually remove it from preferences
         settings.rtmpUrlsList = newList.toSet()
     }
-}
\ No newline at end of file
+}
+
+// this is only required for hms beam bots, your beam auth mechanism will likely be different.
+fun String.meetingToHlsUrl() =
```

Comment:
we can have it as a generic method in Constants , this can be used if user passes preview for meeting url too

Reply:
This kind of link is only required to start hls and if it was a preview link it would still work, the replace would be a no-op.